Sources/Network/HTTPS/HttpConnection/HttpConnection.swift

```diff
@@ -81,13 +84,16 @@ extension HttpConnection {
         func processResponse<Response>(callResult: HttpCallResult<Response>)
             -> Result<Response, ConnectionError>
         {
-            
+
             guard let status = callResult.status else {
+                let message = "Invalid parameters, request not made."
                 return .failure(.connectionFailure(
-                    ["Invalid parameters, request not made.",
-                     callResult.error?.localizedDescription].compactMap({$0}).joined(separator: " ")))
+                    [message, callResult.error?.localizedDescription]
+                     .compactMap({ $0 })
+                     .joined(separator: " "))
```

Comment:
Similar to your other code changes, since .connectionFailure is nested, the next line should be double-indented, and the functional calls should be indented once from there too, like this maybe?
```suggestion
                        [message, callResult.error?.localizedDescription]
                            .compactMap({ $0 })
                            .joined(separator: " "))

```

Makes the whole block look like this:
```
            guard let status = callResult.status else {
                let message = "Invalid parameters, request not made."
                return .failure(.connectionFailure(
                        [message, callResult.error?.localizedDescription]
                            .compactMap({ $0 })
                            .joined(separator: " "))
                )
            }
```